<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Grafico</class>
 <widget class="QDialog" name="Grafico">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1250</width>
    <height>927</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* Estilo elegante con gama de marrones - Versión mejorada */

/* Estilo general para widgets */
QWidget {
    background-color: #E8DCCA; /* Fondo en tono marrón claro en lugar de blanco */
    color: #4A3B24;
    font-family: &quot;Segoe UI&quot;, Arial, sans-serif;
}

/* Botones con efecto 3D */
QPushButton {
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                      stop:0 #A38269, stop:1 #8B6E4E);
    color: #FFFFFF;
    border-radius: 4px;
    border: 1px solid #6D563D;
    padding: 6px 12px;
    font-weight: bold;
    min-height: 20px;
}

QPushButton:hover {
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                      stop:0 #B99980, stop:1 #A38269);
}

QPushButton:pressed {
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                      stop:0 #6D563D, stop:1 #8B6E4E);
    padding-top: 7px;
    padding-bottom: 5px;
}

/* Entradas de texto */
QLineEdit, QTextEdit, QPlainTextEdit {
    background-color: #F5F0EB; /* Más claro que el fondo general */
    border: 1px solid #C4A484;
    border-radius: 3px;
    padding: 3px;
    selection-background-color: #C4A484;
    selection-color: #FFFFFF;
}

/* ComboBox */
QComboBox {
    background-color: #F5F0EB;
    border: 1px solid #C4A484;
    border-radius: 3px;
    padding: 3px 18px 3px 3px;
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 15px;
    border-left: 1px solid #C4A484;
}

/* Menús */
QMenuBar {
    background-color: #D0BFA9;
}

QMenuBar::item {
    background: transparent;
    padding: 4px 10px;
}

QMenuBar::item:selected {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                               stop:0 #A38269, stop:1 #8B6E4E);
    color: #FFFFFF;
}

/* Barras de desplazamiento */
QScrollBar:vertical {
    border: none;
    background: #D0BFA9;
    width: 10px;
    margin: 0px;
}

QScrollBar::handle:vertical {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
                               stop:0 #8B6E4E, stop:1 #A38269);
    min-height: 20px;
    border-radius: 5px;
}

/* Tabs con efecto 3D */
QTabWidget::pane {
    border: 1px solid #C4A484;
    background-color: #E8DCCA;
}

QTabBar::tab {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                               stop:0 #D0BFA9, stop:1 #C4A484);
    padding: 6px 12px;
    border: 1px solid #C4A484;
    border-bottom: none;
    border-top-left-radius: 4px;
    border-top-right-radius: 4px;
}

QTabBar::tab:selected {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                               stop:0 #A38269, stop:1 #8B6E4E);
    color: #FFFFFF;
}

/* Tablas */
QTableView {
    background-color: #F0E6D9;
    gridline-color: #C4A484;
    selection-background-color: #C4A484;
    selection-color: #FFFFFF;
    alternate-background-color: #E8DCCA;
}

QHeaderView::section {
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                     stop:0 #A38269, stop:1 #8B6E4E);
    color: #FFFFFF;
    padding: 4px;
    border: 1px solid #6D563D;
}

/* Grupos */
QGroupBox {
    border: 1px solid #C4A484;
    border-radius: 5px;
    margin-top: 20px;
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                     stop:0 #E8DCCA, stop:1 #D0BFA9);
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top center;
    padding: 0 5px;
    color: #4A3B24;
    font-weight: bold;
}

</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="2" column="3">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>     Salir      </string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="QLabel" name="graficoRueda">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">/* Estilo para QLabel con imagen*/
QLabel {
border: 3px solid #C4A484; /*Borde marrón claro*/
border-radius: 8px; /* Bordes redondeados */
background-color: #F5F0EB; /*Fondo claro para resaltar */
padding: 0px; /* Espaciado interno */
}</string>
     </property>
     <property name="text">
      <string>imagen</string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="5">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="7">
      <widget class="QComboBox" name="comboBox_3"/>
     </item>
     <item row="1" column="5">
      <widget class="QComboBox" name="comboBox_2"/>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboBox_9"/>
     </item>
     <item row="0" column="7">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Amigos</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Dinero</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QComboBox" name="comboBox_6"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Amor</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="8">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Crecimiento</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Espiritualidad</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Fisico</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="6">
      <widget class="QComboBox" name="comboBox_5"/>
     </item>
     <item row="1" column="0">
      <widget class="QComboBox" name="comboBox"/>
     </item>
     <item row="1" column="2">
      <widget class="QComboBox" name="comboBox_8"/>
     </item>
     <item row="0" column="5">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Familia</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QComboBox" name="comboBox_7"/>
     </item>
     <item row="0" column="6">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Proposito</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="8">
      <widget class="QComboBox" name="comboBox_4"/>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Mente</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>Exportar</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="4">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
